decidim/decidim
# Add colors accessibility warning in admin Appearance

#### :tophat: What? Why?

This PR was originally made by @IvanOrtizp, as I didn't get any feedback I made this PR.

> Add a alert message regarding the implications on accessibility of the website because color picking. It should say something about WAI contrast.

Thanks, @IvanOrtizp for the initial implementation!!

#### :pushpin: Related Issues

- Supersedes  #7931
- Fixes #6543

#### Testing

1. Sign in as admin
2. Go to appearance in admin panel 
3. See warning message 

### :camera: Screenshots

![imatge](https://user-images.githubusercontent.com/717367/134887873-b9732d4a-1861-465f-ab85-d2ab4d25ce39.png)


:hearts: Thank you!
